src/workloads/datasets/names.txt

```diff
@@ -0,0 +1,2000 @@
+JAMES
```

Comment:
This and the previous are really cool. Three questions: 
1. Can we document these datasets somewhere? 
2. Should they be under workloads or in some reference directory? 
3. Why are all the names all caps?

Reply:
1. I have added the documentation to `using.md`.
2. I think that since this is part of a generator, workloads/datasets makes more sense. But let me know if you have any other idea.
3. Because this is how they are distributed. The datasets for names and familiynames are from https://github.com/danielmiessler/SecLists. In any case, I modified both datasets so instead of JAMES it will be James.